ci/workflows/smoke.workflow

```diff
@@ -0,0 +1,28 @@
+workflow "smoke tests" {
+  resolves = "run test"
+}
+
+action "build ceph and skyhook cls" {
+  uses = "popperized/cmake@ubuntu-18.04"
+  args = "cls_tabular run-query ceph_test_skyhook_query vstart"
+  env = {
+    CMAKE_PROJECT_DIR = "./"
+    CMAKE_INSTALL_DEPS_SCRIPT = "install-deps.sh"
```

Comment:
We might need to put the `install-deps.sh` binaries in a container and pull that instead. :( 

I constantly had to fight the 1 hour time limit to get this work.